### PR TITLE
Update README.

### PR DESCRIPTION
**Pull request by ZekeMedley**
This is a mostly aesthetic improvement to the README. 

For now as least, I tried to to stay away from adding any additional information. Down the line we might want to discuss adding a little more, particularity around the getting started section.

"Pretty" version here: https://github.com/zeek/zeek/blob/topic/zeke/README/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+`git clone --recursive https://github.com/zeek/zeek `
+
+Then, with its [dependencies](https://docs.zeek.org/en/stable/install/install.html#prerequisites) installed, build and install.
+
+`./configure && make && sudo make install`
+
+
+## Development
+Zeek was originally developed by Vern Paxson. Robin Sommer now leads the project, jointly with a core team of researchers and developers at the [International Computer Science Institute](http://www.icsi.berkeley.edu) in Berkeley, CA; and the [National Center for Supercomputing Applications](http://www.ncsa.illinois.edu) in Urbana-Champaign, IL.
```
Comment (rsmmr) [inline README.md]: Let's just skip this paragraph, it's not up to date and generally a bit awkward to begin with. I checked with Vern, and he's ok with removing it, too.

**Comment by ZekeMedley** [comment]
I think it's worthwhile to add a community section to this as well. I pinged @akgraner and we'll work on putting something together.

**Comment by jsiwek** [comment]
* I'd keep `README` as the canonical file name.  Note the Travis build currently fails because it's now removed and we have a CMake packaging script rely on it, but who know if there's other external package maintainers that also are relying on `README`.

* Feels weird to change to Markdown from reStructuredText for only this one thing since the later is used everywhere else.  Is the change just for the embedded HTML or something more?  Does GH allow the "raw" directive in reST to do the equivalent embedding ?

**Comment by rsmmr** [comment]
Ah, I didn't see the HTML, was looking only at the "pretty" version so far. I'd rather avoid raw HTML, that looks odd when not rendered -- which for a README should still be a common way to look at it.

**Comment by ZekeMedley** [comment]
The change is just so that I can used the embedded HTML. That offers a lot more flexibility when its being rendered. I'll take a stab with reStructuredText though. I'm not sure that will look quite as nice (particularly the table of contents), but it makes sense that we try to avoid the HTML.

**Comment by ZekeMedley** [comment]
I spent some time with both reStructuredText and Markdown but neither have a way to center items which I think is a fairly important visual improvement that this brings. I do think that the HTML tags can be arranged in such a way that they're fairly unobtrusive so I've gone ahead and done that instead and also tried to format the plaintext so that it is fairly readable.

Let me know what you think :)

**Comment by 0xxon** [comment]
Just as a discussion point - might it make sense to just have 2 readme files? One that is the md/rest with the html tags, and one that is just rendered to text, excluding the html?

That way a normal user on the command line will probably just look at the (text only) readme - and from what I remember github will automatically render the .md/rest instead.

**Comment by ZekeMedley** [comment]
I'm a fan of that approach @0xxon. Its less maintainable but, historically, our README hasn't been updated too often.

**Comment by rsmmr** [comment]
There's an update to the `cmake` submodule in there. Is that intentional? The commit isn't available to fetch.

**Comment by ZekeMedley** [comment]
That was an accident. Thanks for merging this in :)

**Pull request merged**